server/aws/resources/README.md

```
@@ -0,0 +1,193 @@
+# AWS Resources
+
+CDK stack for vcknots on AWS.
+
+Related package: [`@trustknots/aws`](../provider) (AWS providers for DynamoDB / KMS / Secrets Manager — placeholder).
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)